packages/pacman/build.sh

```diff
@@ -3,7 +3,7 @@ TERMUX_PKG_DESCRIPTION="A library-based package manager with dependency support"
 TERMUX_PKG_LICENSE="GPL-2.0"
 TERMUX_PKG_MAINTAINER="@Maxython"
 TERMUX_PKG_VERSION=6.0.1
-TERMUX_PKG_REVISION=4
+TERMUX_PKG_REVISION=5
 TERMUX_PKG_SRCURL=https://sources.archlinux.org/other/pacman/pacman-${TERMUX_PKG_VERSION}.tar.xz
 TERMUX_PKG_SHA256=0db61456e56aa49e260e891c0b025be210319e62b15521f29d3e93b00d3bf731
 TERMUX_PKG_DEPENDS="bash, libarchive, curl, gpgme"
```

Comment:
Also add openssl as a dependency here to prevent such issues from happening in future

Reply:
ok